specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1469 @@
+        },
+        "lastActionDateTimeUtc": {
+          "format": "date-time",
+          "description": "Date time in which the operation's status has been updated",
+          "type": "string",
+          "example": "2020-01-20T11:41:07.7010000-08:00"
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        },
+        "summary": {
+          "$ref": "#/definitions/StatusSummary"
+        }
+      }
+    },
+    "OperationStatusResponse": {
+      "description": "Document Status Response",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The summary status of individual operation",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationStatusSummary"
+          }
+        },
+        "@nextLink": {
+          "description": "Url for the next page.  Null if no more pages available",
+          "type": "string",
+          "example": "https://westus.cognitiveservices.azure.com/translator/text/batch/v1.0.preview.1/operations?$top=5&$skip=15"
+        }
+      }
+    },
+    "DocumentSubmissionResponse": {
+      "description": "The response for a document submission",
+      "type": "object",
+      "properties": {
+        "job": {
+          "$ref": "#/definitions/OperationStatusSummary"
+        },
+        "documentDownloadUrl": {
+          "description": "Document download link",
+          "type": "string"
+        },
+        "documentExpirationDateTimeUtc": {
+          "format": "date-time",
+          "description": "Datetime in Utc when the document will no longer be available",
+          "type": "string"
+        }
+      }
+    },
+    "DocumentStatusDetail": {
+      "required": [
+        "createdDateTimeUtc",
+        "lastActionDateTimeUtc",
+        "path",
+        "status",
+        "to"
+      ],
+      "type": "object",
+      "properties": {
+        "path": {
+          "description": "Location of the document or folder",
+          "type": "string",
+          "example": "https://myblob.blob.core.windows.net/mycontainer/fr/mydoc.txt"
+        },
+        "createdDateTimeUtc": {
+          "format": "date-time",
+          "description": "Operation created date time",
+          "type": "string",
+          "example": "2020-01-20T11:40:07.7010000-08:00"
+        },
+        "lastActionDateTimeUtc": {
+          "format": "date-time",
+          "description": "Date time in which the operation's status has been updated",
+          "type": "string",
+          "example": "2020-01-20T11:41:07.7010000-08:00"
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        },
+        "detectedLanguage": {
+          "description": "Detected language of the original document (to be implemented)",
+          "type": "string",
+          "example": "en"
+        },
+        "to": {
+          "description": "To language",
+          "type": "string"
+        },
+        "errorMessage": {
```

Comment:
In other Cognitive APIs, errorCode and errorMessage are grouped together, and in some way related to the ErrorV2 parts.  See TextAnalytics as an example: https://github.com/Azure/azure-rest-api-specs/blob/328f2733e25ace387d21a739f3d21070eebf97e4/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/TextAnalytics.json#L385

Reply:
changed to ErrorV2 format.